src/shared/scrapers/BE/index.js

```diff
@@ -0,0 +1,136 @@
+
+    const dataByRegion = {};
+    const dataByProvince = {};
+    let nationalData = { tested: 0 };
+
+    for (const item of casesData) {
+      if (item.DATE === 'NA' || datetime.dateIsBeforeOrEqualTo(item.DATE, date)) {
+        if (!dataByProvince[item.REGION]) {
+          dataByProvince[item.REGION] = {};
+        }
+        const regionData = dataByProvince[item.REGION];
+
+        if (!regionData[item.PROVINCE]) {
+          regionData[item.PROVINCE] = {};
+        }
+        const provinceData = regionData[item.PROVINCE];
+
+        provinceData.cases = parse.number(item.CASES) + (provinceData.cases || 0);
+      }
+    }
+
+    for (const item of deathsData) {
+      if (item.DATE === 'NA' || datetime.dateIsBeforeOrEqualTo(item.DATE, date)) {
+        if (!dataByRegion[item.REGION]) {
+          dataByRegion[item.REGION] = {};
+        }
+        const regionData = dataByRegion[item.REGION];
+
+        regionData.deaths = parse.number(item.DEATHS) + (regionData.deaths || 0);
+      }
+    }
+
+    for (const item of hospitalizedData) {
+      if (item.DATE === 'NA' || datetime.dateIsBeforeOrEqualTo(item.DATE, date)) {
+        if (!dataByProvince[item.REGION]) {
+          dataByProvince[item.REGION] = {};
+        }
+        const regionData = dataByProvince[item.REGION];
+
+        if (!regionData[item.PROVINCE]) {
+          regionData[item.REGION] = {};
+        }
+        const provinceData = regionData[item.PROVINCE];
+
+        provinceData.hospitalized = parse.number(item.NEW_IN) + (provinceData.hospitalized || 0);
+        provinceData.discharged = parse.number(item.NEW_OUT) + (provinceData.discharged || 0);
+      }
+    }
+
+    for (const item of testsData) {
+      if (item.DATE === 'NA' || datetime.dateIsBeforeOrEqualTo(item.DATE, date)) {
+        nationalData.tested += parse.number(item.TESTS);
+      }
+    }
+
+    const data = [];
+
+    for (const reg of Object.keys(dataByRegion)) {
+      let regionData = {
+        state: mappings[reg],
+        ...dataByRegion[reg]
+      };
+
+      if (reg === 'Flanders' || reg === 'Wallonia') {
+        const provinceData = [];
+        for (const prov of Object.keys(dataByProvince[reg])) {
+          provinceData.push({
+            state: mappings[reg],
+            county: mappings[prov],
+            ...dataByProvince[reg][prov]
+          });
+        }
+
+        regionData = transform.sumData(provinceData, regionData);
+        data.push(...provinceData);
+      } else if (reg === 'Brussels' || reg === 'NA') {
+        regionData = {
+          ...dataByProvince[reg][reg],
+          ...regionData
+        };
+      }
+
+      nationalData = transform.sumData([regionData], nationalData);
+      data.push(regionData);
+    }
+
+    data.push(nationalData);
+
+    for (const item of data) {
+      assert(iso1Codes[item.country.replace('iso1:', '')], `Missing iso1 code for country ${item.country}`);
```

Comment:
Error processing iso1:BE:  TypeError: Cannot read property 'replace' of undefined
    at Object.scraper (/Users/user/nonsync/dev/newcases-org/covidatlas/src/shared/scrapers/BE/index.js:124:37)